moped-api/Dockerfile

```diff
@@ -3,6 +3,9 @@ EXPOSE 5000
 # We copy just the requirements.txt first to leverage Docker cache
 COPY requirements/development.txt /app/requirements.txt
 
+RUN apt-get update
+RUN apt-get install -y gcc python3-dev
+
```

Comment:
I was using the dockerfile to test why things were breaking, turns out it was unrelated to the code, but when debugging, the docker image wouldnt build b/c

```#8 19.79       creating build/temp.linux-aarch64-3.8/psutil
#8 19.79       gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_SIZEOF_PID_T=4 -DPSUTIL_VERSION=580 -DPSUTIL_LINUX=1 -DPSUTIL_ETHTOOL_MISSING_TYPES=1 -I/usr/local/include/python3.8 -c psutil/_psutil_common.c -o build/temp.linux-aarch64-3.8/psutil/_psutil_common.o
#8 19.79       unable to execute 'gcc': No such file or directory
#8 19.79       C compiler or Python headers are not installed on this system. Try to run:
#8 19.79       sudo apt-get install gcc python3-dev
#8 19.79       error: command 'gcc' failed with exit status 1
#8 19.79       [end of output]
#8 19.79   
#8 19.79   note: This error originates from a subprocess, and is likely not a problem with pip.
#8 19.79 error: legacy-install-failure
#8 19.79 
#8 19.79 × Encountered error while trying to install package.
#8 19.79 ╰─> psutil
```
so I had to add this for it to work

Reply:
somewhat related, I first tried following the steps in the readme to run it locally in a virtualenv and kept running into issues installing the `cryptography` package. python.h not found issues.